src/pages/high/ros.md

```diff
@@ -0,0 +1,139 @@
+# Luos with ROS1 and ROS2
+
+Luos comes with a package for the [Robot Operating System](https://www.ros.org/).
+
+ROS 2 is the default version but backward compatibility with ROS 1 is ensured via the official ROS 1 bridge. You can get an example of an application using Luos modules in ROS 2 with the [bike sharing example](https://github.com/aubrune/luos_bike_alarm_example).
+
+In this tutorial, we will assume you're using ROS 2. If you want to communicate with a ROS 1 ecosystem, follow this quickstart anyway since ROS 2 needs to be installed and then refer to this [ROS 1 guide](./ros/ros1.md).
```

Comment:
```suggestion
In this tutorial, we will assume you're using ROS 2. If you want to communicate with a ROS 1 ecosystem, follow this quickstart anyway since ROS 2 needs to be installed and then refer to the [Retrocompatibility with ROS 1](./ros/ros1.md) page.
```